demo/index.html

```diff
@@ -15,226 +15,213 @@
+
+          <div class="example without-preload">
+            <button onclick="load('example-without-preload-1')">
+              Load image
+            </button>
+            <br>
+            <iron-image sizing="contain" alt="The Polymer logo." id="example-without-preload-1"></iron-image>
+          </div>
+        </template>
+      </demo-snippet>
+
+      <h3><code>preload</code> Attribute with Placeholder Color</h3>
+      <demo-snippet class="centered-demo">
+        <template>
+          <style is="custom-style">
+            .example.preload iron-image {
+              width: 150px;
+              height: 150px;
+              background: #ddd;
+              --iron-image-placeholder: {
+                background: #939ed5;
+              };
+            }
+          </style>
+
+          <div class="example preload">
+            <button onclick="load('example-preload-1')">
+              Load image
+            </button>
+            <br>
+            <iron-image preload id="example-preload-1" alt="The Polymer logo." class="sized" sizing="contain"></iron-image>
+            <br>
+            (without <code>fade</code> attribute)
+          </div>
+          <div class="example preload">
+            <button onclick="load('example-preload-2')">
+              Load image
+            </button>
+            <br>
+            <iron-image preload fade id="example-preload-2" alt="The Polymer logo." class="sized" sizing="contain"></iron-image>
+            <br>
+            (with <code>fade</code> attribute)
+          </div>
+        </template>
+      </demo-snippet>
+
+      <h3><code>preload</code> Attribute with Placeholder Image</h3>
+      <demo-snippet class="centered-demo">
+        <template>
+          <style is="custom-style">
+            .example.preload-image iron-image {
+              width: 150px;
+              height: 150px;
+              background: #ddd;
+            }
+          </style>
+
+          <div class="example preload-image">
+            <button onclick="load('example-preload-image-1')">
+              Load image
+            </button>
+            <br>
+            <iron-image preload placeholder="./loading.png" id="example-preload-image-1" alt="The Polymer logo." class="sized" sizing="contain"></iron-image>
+            <br>
+            (without <code>fade</code> attribute)
+          </div>
+          <div class="example preload-image">
+            <button onclick="load('example-preload-image-2')">
+              Load image
+            </button>
+            <br>
+            <iron-image preload placeholder="./loading.png" fade id="example-preload-image-2" alt="The Polymer logo." class="sized" sizing="contain"></iron-image>
+            <br>
+            (with <code>fade</code> attribute)
+          </div>
+        </template>
+      </demo-snippet>
+
+      <h3>(Used by all demos.)</h3>
+      <demo-snippet class="demo-hidden">
+        <template>
+          <style>
+            .example {
```

Comment:
I think you can just move this into the main page style and not worry about it :)